### DISCUSSION TITLE
Deployments stuck in "Queued"

### DISCUSSION THREAD

**Discussion by mitraraman** [vercel/community#2848] [Bug Report]
### Page to Investigate

chiyoadmin-1imi0vjgq-mitraraman.vercel.app

### Steps to Reproduce

Pushed a change to GH and the deployment shows up in Vercel but is stuck in Queued for 5+ minutes now. I tried to redeploy and that also stayed in Queued status. Any suggestions for how to get it to deploy? Is there a Vercel issue?

**Comment by mitraraman** [comment]
Looks like it now failed because of a network timeout. Error message:

network timeout at: https://api-iad1.vercel.com/v2/now/deployments/dpl_H2WMdL8GvWE4sVvkMJZDhGSf7oXL/builds/bld_j8ths9lkq/lambda?ownerId=kQaqijiFhVfNANZmur13MgBa&projectId=prj_nxC21Qw2pJIf2SyHNndPwCjtt9Qu&traceCarrier=%7B%22ot-baggage-webhookAt%22%3A%221686684320549%22%2C%22ot-baggage-senderUsername%22%3A%22gh.mitraraman%22%2C%22x-datadog-trace-id%22%3A%222102390310448627247%22%2C%22x-datadog-parent-id%22%3A%226839325529666269111%22%2C%22x-datadog-sampling-priority%22%3A%222%22%2C%22traceparent%22%3A%2200-00000000000000001d2d30d931837e2f-5eea2a553b04afb7-01%22%2C%22tracestate%22%3A%22dd%3Ds%3A2%22%7D&traceCarrier=%7B%22ot-baggage-webhookAt%22%3A%221686684320549%22%2C%22ot-baggage-senderUsername%22%3A%22gh.mitraraman%22%2C%22x-datadog-trace-id%22%3A%222102390310448627247%22%2C%22x-datadog-parent-id%22%3A%228809165415351095970%22%2C%22x-datadog-sampling-priority%22%3A%222%22%2C%22traceparent%22%3A%2200-00000000000000001d2d30d931837e2f-7a407145b5a892a2-01%22%2C%22tracestate%22%3A%22dd%3Ds%3A2%22%7D&traceCarrier=%

**Comment by amyegan** [comment]
There is an open incident related to serverless functions: https://www.vercel-status.com/incidents/cqkdqrzqkspm

Can switch your region from `iad1` to `cle1` redeploy? Please let me know if that solves the problem for you 🙏

**Comment by mitraraman** [comment]
Worked! Thank you!
